UI/FleetButton.cpp

```diff
@@ -342,10 +342,12 @@ void FleetButton::LayoutIcons() {
             }
         }
 
-        if (available_exits_count >= 1)
+        if (fleet->Owner() == ALL_EMPIRES)  // as above, if first fleet of fleet-button is "monster-owned", all fleets are
+            SetBrowseText(UserString("FB_TOOLTIP_BLOCKADE_MONSTER"));
+        else if (available_exits_count >= 1)
             SetBrowseText(UserString("FB_TOOLTIP_BLOCKADE_WITH_EXIT") + available_exits);
         else
-            SetBrowseText(UserString("FB_TOOLTIP_BLOCKADE_NO_EXIT"));
+            SetBrowseText(UserString("FB_TOOLTIP_BLOCKADE_NO_EXIT")); // this currently shouldn't happen, empire ships should always have an exit
```

Comment:
For now, I've removed the comment, but I've made a note of it and I'll report when I was able to test if a fleet can indeed have no exits if they are newly produced or arrive by star gate etc..